### DISCUSSION TITLE
does seaport support amoy testnet?

### DISCUSSION THREAD

**Discussion by Ayejang** [ProjectOpenSea/seaport#1320] [Q&A] [answered]
I've saw [amoy support announcement](https://docs.opensea.io/changelog/polygon-amoy-testnet) today. 

but the seaport github readme docs is not updated, and when I try to execute createConduit on amoy chain to seaport 1.1 addressess, it returns nothing.

Does seaport support amoy? If not, I want to know when it will be.

**Comment by ryanio** [comment]
You may have to deploy seaport 1.1 yourself, OpenSea is now on Seaport 1.6 so that is what we've deployed to Amoy. I'll update the readme today.

**Reply by ryanio** [reply, marked as answer]
i deployed seaport 1.1 on amoy for you, and have opened a PR to update the readme

**Reply by Ayejang** [reply]
thank you so much!
